qa/tasks/radosgw_admin.py

```diff
@@ -1026,9 +1026,51 @@ def task(ctx, config):
         stdin=StringIO(json.dumps(out)),
         check_status=True)
 
-    if realm is None:
-        (err, out) = rgwadmin(ctx, client, ['zone', 'get','--rgw-zone','default'])
-    else:
-        (err, out) = rgwadmin(ctx, client, ['zone', 'get'])
+    (err, out) = rgwadmin(ctx, client, ['zone', 'get'])
     assert len(out) > 0
     assert len(out['placement_pools']) == orig_placement_pools + 1
+
+    zonecmd = ['zone', 'placement', 'rm',
+	'--rgw-zone', 'default',
+	'--placement-id', 'new-placement']
+
+    (err, out) = rgwadmin(ctx, client, zonecmd, check_status=True)
+
+    # TESTCASE 'zonegroup-info', 'zonegroup', 'get', 'get zonegroup info', 'succeeds'
+    (err, out) = rgwadmin(ctx, client, ['zonegroup', 'get'], check_status=True)
```

Comment:
this was the last line in the original diff - i'm not sure we want to include the remainder

Reply:
@cbodley made the changes. Kindly review it.